docs/source/conf-schemas.rst

```diff
@@ -290,6 +307,70 @@ To extract these nested records, use the ``configuration`` option ``json_path``:
     }
   }
 
+Log Patterns
+~~~~~~~~~~~~
+
+Log patterns provide the ability to differentiate log schemas that are identical or very close in nature.
+
+They can be added by using the ``configuration`` option ``log_patterns``.
+
+Log patterns are a collection of key/value pairs where the key is the name of the field, and the value is a list of
+expressions to search for in said field of the log. The log parsers will compare the values provided as ``log_patterns``
+for a field to the value contained in the corresponding field of the log. If *any* of the log patterns listed exists in
+a specific field, the parser will consider the data valid.
+
+This feature is especially helpful to reduce false positives, since it provides to ability to only match a schema if
+specific values are present in a log.
+
+Wild card log patterns are supported using the ``*`` or ``?`` symbols, as shown below::
+
+Example schema::
+
+  {
+    "log_name": {
+      "schema": {
+        "computer_name": "string",
+        "hostname": "string",
+        "instance_id": "string",
+        "process_id": "string",
+        "message": "string",
+        "timestamp": "float",
+        "type": "string"
+      },
+      "parser": "json",
+      "configuration": {
+        "log_patterns": {
+          "type": [
+            "*bad.log.type*"
+          ]
+        }
+      }
+    }
+  }
+
+Example logs::
+
+  {
+    "computer_name": "test-server-name",
+    "hostname": "okay_host",
+    "instance_id": "95909",
+    "process_id": "82571",
+    "message": "this is not important info"
+    "timestamp": "1427381694.88",
+    "type": "good.log.type.value"             # this will not match the schema above
```

Comment:
I'd say `match the configuration` above, since the schema doesn't have `log_patterns` as an option